app/models/backend_api.rb

```diff
@@ -25,6 +25,7 @@ class BackendApi < ApplicationRecord
 
   alias_attribute :api_backend, :private_endpoint
 
+  # before_validation :set_private_endpoint, :set_port_private_endpoint
```

Comment:
Why do we need this line twice but one of them commented?

Reply:
mistake 😄 .. will remove it